Bio/Wise/psw.py

```diff
@@ -85,10 +86,11 @@ def __init__(self, unit, column, kind):
         self.column = column
         self.kind = kind
 
-    def __str__(self):
+    def __repr__(self):
+        """Represent the ColumnUnit object as a string for debugging."""
         return "ColumnUnit(unit=%s, column=%s, %s)" % (self.unit, self.column, self.kind)
 
-    __repr__ = __str__
+    __str__ = __repr__
```

Comment:
I think this line can be omitted? (Given that what I commented above is true)

Reply:
Yes, removed.